docs/deployment_guide/partner_editable/troubleshooting.adoc

```diff
@@ -96,18 +96,24 @@ Verify that an RSA key pair used during deployment. If not, try the following op
 // git log --reverse origin/main...v1.4.0 -- ':!/ci/'
 
 // Changelog links
+:imds-v2: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html
+:tap-1_4_2-cl: https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/release-notes.html#v142-0
 :tap-1_4_1-cl: https://docs.vmware.com/en/VMware-Tanzu-Application-Platform/1.4/tap/release-notes.html#v141-0
 :ce-1_4_1-cl: https://docs.vmware.com/en/Cluster-Essentials-for-VMware-Tanzu/1.4/cluster-essentials/release-notes.html#v141-0
 :eks-1_24-cl: https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-1.24
 :eksqs-5_0_0-cl: https://github.com/aws-quickstart/quickstart-amazon-eks/releases/tag/v5.0.0
 
+* Version 1.4.2
```

Comment:
we added a variable for the tap version here: https://github.com/vmware-tap-on-public-cloud/quickstart-vmware-tanzu-application-platform/blob/a0932cc5838edc42a1c6b028a50d522925869a45/docs/deployment_guide/partner_editable/_settings.adoc#L9 

so that we ideally only have to change this in one place at some time. so it would be like {partner-product-version} to replace 1.4.2

Reply:
I don't think we want to be "dynamic" on the release notes, as this is a static record of history.

Reply:
FWIW: the variables for the link above are just about making it (subjectively) easier to write the docs, what I was going for is something similar to [reference style links from the markdown world](https://www.markdownguide.org/basic-syntax/#reference-style-links)

Reply:
I recommend keeping a log of all RNs. 
static history is fine as this is a one-time activity.